ModulemdTranslationHelpers/Utils.py

```diff
@@ -1,14 +1,136 @@
 import sys
+import logging
+import gi
+import requests
+import koji
 from babel.messages import Catalog, pofile
 from datetime import datetime
 from collections import defaultdict
 
-try:
-    import gi
-    gi.require_version('Modulemd', '2.0')
-    from gi.repository import Modulemd
-except ImportError:
-    sys.exit(77)
+gi.require_version('Modulemd', '2.0')  # noqa
+from gi.repository import Modulemd
+
+
+class KojiSessionMock:
+
+    def __init__(self):
+        with open('../modulemd/tests/test_data/f29.yaml', 'r') as file:
```

Comment:
> A proper test here would return just a single module stream document per build_id

I didn't quite understand this bit. Am I not assigning a document only after receiving the `build_id`?

Reply:
I mean that you shouldn't use f29.yaml for this test, because we actually want the lookups to give us back individual module stream objects (so a single-document YAML) rather than the complete repodata.

Reply:
@sgallagher 
So I have created a new test file which has the sum of 2 individual module stream objects. I hope this is what you meant!

Reply:
I've added 2 module stream objects of `f29` that happen to be the first 2 strings in all the `.po` files.